packages/angular/projects/ui-angular/src/lib/components/amplify-force-new-password/amplify-force-new-password.component.html

```diff
@@ -11,8 +11,14 @@
       <amplify-form-input
         name="password"
         type="password"
-        label="Change password"
-        autocomplete="password"
+        autocomplete="new-password"
```

Comment:
nit - do we need a label in this `input` as well?

Reply:
No, I have it so that it infers label automatically from from the input name:

https://github.com/aws-amplify/amplify-ui/blob/911c82c2c7e2985bd0b8a6d4cbbe92a2fde7b53d/packages/ui/src/helpers/auth.ts#L38

This will be cleaned up with Angular i18n coming this week.

Reply:
We can provide a specific `label` value if we want to override the default label, such as "Confirm Password".